perimysium/dataman.py

```diff
@@ -409,6 +409,18 @@ def write(self, fpath):
 
         f.close()
 
+    def addnoise(self, noiseWidth):
+        for imarker in range(self.num_markers):
+
+            # create the noise object
+            noise = np.random.normal(0, noiseWidth, self.num_frames)
+            # add noise the marker existing marker data.
+            self.data[self.marker_names[imarker] + '_tx'] += noise
```

Comment:
Perhaps the noise added to each coordinate should be different? Also, "the marker" should be "to the" in the comment. :horse: